gulpfile.js

```diff
@@ -106,21 +106,36 @@ gulp.task('compile_test', function() {
 });
 
 gulp.task('copyTypings', function() {
-  return gulp.src(['src/index.d.ts', 'src/firebase-namespace.d.ts'])
+  return gulp.src([
+    'src/index.d.ts',
+    'src/firebase-namespace.d.ts',
+    'src/**/protos/*.d.ts',
+  ])
     // Add header
     .pipe(header(banner))
     .pipe(gulp.dest(paths.build))
 });
 
-gulp.task('compile_all', gulp.series('compile', 'copyTypings', 'compile_test'));
+gulp.task('copyJSON', function() {
+  return gulp.src([
+    'src/**/protos/*.json',
```

Comment:
I think this is included in the next pattern, and therefore is probably not needed.

Reply:
Good point, updated.